fbw-common/src/wasm/cpp-msfs-framework/lib/inih/ini_type_conversion.h

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2023 FlyByWire Simulations
+// SPDX-License-Identifier: GPL-3.0
```

Comment:
As above.

Reply:
This might actually be a file we created (prob AGuther) - will double check

Reply:
Pretty sure this was done by Andreas